llpc/context/llpcCompiler.cpp

```diff
@@ -2271,6 +2277,46 @@ void helperThreadBuildRayTracingPipelineElf(IHelperThreadProvider *helperThreadP
   helperThreadPayload->compiler->releaseContext(context);
 }
 
+// =====================================================================================================================
+// Limited implementation of Llpc::IHelperThreadProvider to support -add-rt-helpers.
+//
+// If no deferred work helper thread providers is available when additional threads are requested via -add-rt-helpers
+// then use an instances of this class to coordinate helper threads.
+class InternalHelperThreadProvider : public Llpc::IHelperThreadProvider {
+public:
+  InternalHelperThreadProvider() {}
+  ~InternalHelperThreadProvider() {}
```

Comment:
Nit: use `=default` to avoid the user-defined constructor/destructor, or just remove those completely.